src/servers/exporter.ts

```diff
@@ -318,3 +320,23 @@ export const getBookingVersusRealWorldAssets = async () => {
     (lightning + bitcoin)
   ) // value in accounting
 }
+
+createGauge({
+  name: "pendingPaymentsCount",
+  description: "How many pending HTLCs there are in the channels of the active nodes",
+  collect: async () => {
+    const pendingPaymentsCount = await getPendingPaymentCount()
+    if (getPendingPaymentCount instanceof Error) return 0
```

Comment:
does grafana understand `NaN`? it's better to return `NaN` versus 0 is that is the case. 

ie: it's better to not show anything that an erroneous value. 

showing a balance of 0 when when maybe it's just lnd that is down is misleading.

Reply:
resolved in [ecda354](https://github.com/GaloyMoney/galoy/pull/2057/commits/ecda354e8a3ad8ac3aefae39163af37a5ff55328)